### PR TITLE
Various test fixes.

### PR DESCRIPTION
**Pull request by oliverchang**
Upgrade GitHub actions image to 20.04 since 16.04 is no longer
available.

Run extra AFL tests that previously weren't being run properly.

(Speculative) Make it easier for the flaky AFL fuzz_test_crash to pass
by adding a corpus with a one-bit-off item.

### CODE REVIEW

File: src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_engine_test.py [hunk 1]
```diff
@@ -118,17 +122,3 @@ def test_fuzz_with_crash(self):
 
     # Testcase (non-zero size) should've been copied back.
     self.assertNotEqual(os.path.getsize(crash.input_path), 0)
-
-  def test_startup_crash_not_reported(self):
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_engine_test.py:122]: I don't really understand this test. @jonathanmetzman is this still relevant? (it was not being run at all before).

**Comment by oliverchang** [comment]
TBR since this is blocking PRs.

File: src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_engine_test.py (hunk 1 above)
Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/tests/core/bot/fuzzers/afl/afl_engine_test.py:122]: I think we don't want AFL to report crashes in corpus. Not 100% sure.

File: local/tests/run_afl_tests [hunk 2]
```diff
@@ -8,4 +8,4 @@ if [ "$(cat /proc/sys/kernel/core_pattern)" != "core" ]; then
 fi
 
 AFL_INTEGRATION_TESTS=1 python butler.py py_unittest \
-    -t core -p '*afl_launcher_integration*' "$@"
+    -t core -p '*afl*' "$@"
```
Comment (jonathanmetzman) [inline local/tests/run_afl_tests:11]: wont this run all afl tests not just the integration tests?
Dont the AFL unittests get run elsewhere?

**Review by jonathanmetzman — COMMENTED** [review]
lgtm

**Pull request merged**